test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -827,19 +827,20 @@ function turbconv_boundary_state!(
     θ_liq_cv,
     q_tot_cv,
     θ_liq_q_tot_cv,
-    tke = subdomain_surface_values(turbconv.surface, turbconv, m, gm, gm_a, zLL)
+    tke =
+        subdomain_surface_values(turbconv.surface, turbconv, m, gm⁻, gm_a⁻, zLL)
 
     @unroll_map(N_up) do i
-        up[i].ρaw = FT(0)
-        up[i].ρa = a_up_surf[i] * gm.ρ
-        up[i].ρaθ_liq = up[i].ρa * θ_liq_up_surf[i]
-        up[i].ρaq_tot = up[i].ρa * q_tot_up_surf[i]
+        up⁺[i].ρaw = FT(0)
+        up⁺[i].ρa = a_up_surf[i] * gm⁻.ρ
+        up⁺[i].ρaθ_liq = up⁺[i].ρa * θ_liq_up_surf[i]
+        up⁺[i].ρaq_tot = up⁺[i].ρa * q_tot_up_surf[i]
```

Comment:
```suggestion
        up⁺[i].ρaθ_liq = up⁻[i].ρa * θ_liq_up_surf[i]
        up⁺[i].ρaq_tot = up⁻[i].ρa * q_tot_up_surf[i]
```

Reply:
the value of "up⁺[i].ρa " was just define in the line above. I would like to make sure we are consistent with the same area in all variables, I will change it to: 

        up⁺[i].ρaw = FT(0)
        up⁺[i].ρa = gm⁻.ρ * a_up_surf[i]
        up⁺[i].ρaθ_liq = gm⁻.ρ * a_up_surf[i] * θ_liq_up_surf[i]
        up⁺[i].ρaq_tot = gm⁻.ρ * a_up_surf[i] * q_tot_up_surf[i]